release/generate_changelog.py

```diff
@@ -0,0 +1,38 @@
+"""Generate changelog entries for all PRs merged since the last release."""
+import requests
+
+get = requests.get('https://api.github.com/repos/RaRe-Technologies/gensim/releases')
+get.raise_for_status()
+
+last_release_date = get.json()[0]['published_at'][:10]
```

Comment:
Is this guaranteed to be sorted? What's `[0]` and `[:10]`? Some code comments please.